docs/examples/loaders/GLTFLoader.html

```diff
@@ -125,7 +125,7 @@
 		[page:ArrayBuffer data] — glTF asset to parse, as an ArrayBuffer or <em>JSON</em> string.<br />
 		[page:String path] — The base path from which to find subsequent glTF resources such as textures and .bin data files.<br />
 		[page:Function onLoad] — A function to be called when parse completes.<br />
-		[page:Function onError] — (optional) A function to be called if an error occurs during parsing. The function receives error as an argument.<br />
+		[page:Function onError] — A function to be called if an error occurs during parsing. The function receives error as an argument.<br />
```

Comment:
I think it would be better to keep both `onError` callbacks optional. Can't have a required parameter after an optional one (`onProgress`) in any case.

Reply:
So, replacing 

    onError( error );

with

    if ( onError ) onError( error );

?

Reply:
👍